e2e/Microsoft.Azure.Devices.E2ETests/TwinE2ETests.cs

```diff
@@ -190,7 +190,7 @@ static public void ClassCleanup()
 
 #if NETCOREAPP2_0
         // GitHub Issue #302
-        [Ignore]
+        //[Ignore]
```

Comment:
Please never comment out Ignore. 
Either remove it together with the TODO and resolve #302 (in which case remove the #if NETCOREAPP2_0 as well) or leave it in.

Reply:
Also, if you could, please change line 192 to // TODO #302 - having a single "TODO" label is easier when searching throughout the code.